ArduCopter/mode_guided.cpp

```diff
@@ -621,10 +621,7 @@ void ModeGuided::set_angle(const Quaternion &attitude_quat, const Vector3f &ang_
     attitude_quat.to_euler(roll_rad, pitch_rad, yaw_rad);
 
     // log target
-    copter.Log_Write_GuidedTarget(guided_mode,
-                           Vector3f{ToDeg(roll_rad), ToDeg(pitch_rad), ToDeg(yaw_rad)} * 100.0,
-                           false,
-                           ang_vel, Vector3f{0.0, 0.0, climb_rate_cms_or_thrust});
+    copter.Log_Write_Guided_Attitude_Target(guided_mode, degrees(roll_rad), degrees(pitch_rad), degrees(yaw_rad), 0, 0, 0, guided_angle_state.thrust, guided_angle_state.climb_rate_cms * 0.01);
```

Comment:
You have lost the angular velocity here (ang_vel). angular_velocity = [roll_rate, pitch_rate, yaw_rate]

Make this pass by const reference instead in the the Logging function.

```suggestion
    copter.Log_Write_Guided_Attitude_Target(guided_mode, degrees(roll_rad), degrees(pitch_rad), degrees(yaw_rad), ang_vel, guided_angle_state.thrust, guided_angle_state.climb_rate_cms * 0.01);
```